documentation/asciidoc/topics/cmd_examples/batch.adoc

```diff
@@ -0,0 +1,3 @@
+create cache sessions --file=/etc/batch/mycache.xml
```

Comment:
@ryanemerson sorry, I know  that's not current PR scope, but I looked in the Batch CR test and can't find embedding files form ConfigMap Data section test to validate that file mounts and picked-up from correct location.

Reply:
Good point. I should probably add a test for that. It does work though, as I'm currently using it for a demo.

Reply:
Excellent, thanks.